sympy/functions/elementary/piecewise.py

```diff
@@ -215,6 +215,8 @@ def _eval_interval(self, sym, a, b):
         mul = 1
         if (a == b) == True:
             return S.Zero
+        elif isinstance((a,b) , Symbol) and (a.atoms(Symbol) or b.atoms(Symbol)):
```

Comment:
I believe ``isinstance((a,b) , Symbol)`` is always ``False`` as a tuple can hardly be a ``Symbol``.